src/sentry/interfaces/exception.py

```diff
@@ -42,14 +42,13 @@ class SingleException(Interface):
     score = 2000
 
     @classmethod
-    def to_python(cls, data, has_system_frames=None, slim_frames=True):
+    def to_python(cls, data, slim_frames=True):
```

Comment:
system frames no longer need storing. We can derive it quickly dynamically. Also turns out this data was invalid for objc and sometimes javascript because processors kick in too late.